spec/index.bs

```diff
@@ -7357,6 +7357,18 @@ dictionary GPUComputePassDescriptor : GPUObjectDescriptorBase {
         </div>
 </dl>
 
+<div class=note>
+Note: The `x`, `y`, and `z` values passed to {{GPUComputePassEncoder/dispatch()}} and
+{{GPUComputePassEncoder/dispatchIndirect()}} are the number of _workgroups_ do dispatch for each
+dimension, *not* the number of shader invocations to perform across each dimension. This matches the
+behavior of modern native GPU APIs, but differs from the behavior of OpenCL.
+
+This means that if a {{GPUShaderModule}} defines an entry point with `@workgroup_size(4, 4)`, and
+work is dispatched to it with the call `computePass.dispatch(8, 8);` the entry point will be invoked
+1024 times total: 4 dispatches of a workgroup size of 8 along both the X and Y axes.
```

Comment:
Should be: 8 dispatches of a workgroup size of 4

Also, I had to re-read this bit a couple of times. Initially I read it as:
"(4 dispatches) of (a workgroup size of 8 along both the X and Y axes)"
which isn't right. Maybe there's a clearer way of phrasing this?

Reply:
Thanks for catching that. Took a crack at re-phrasing it, let me know what you think!